app/utils/cli-install.js

```diff
@@ -89,3 +80,43 @@ exports.addBinToUserPath = () => {
     });
   });
 };
+
+const logNotify = (withNotification, ...args) => {
+  //eslint-disable-next-line no-console
+  console.log(...args);
+  withNotification && notify(...args);
+};
+
+exports.installCLI = withNotification => {
+  if (process.platform === 'win32') {
+    addBinToUserPath()
+      .then(() =>
+        logNotify(
+          withNotification,
+          'Hyper CLI installed',
+          'You may need to restart your computer to complete this installation process.'
+        )
+      )
+      .catch(err =>
+        logNotify(withNotification, 'Hyper CLI installation failed', `Failed to add Hyper CLI path to user PATH ${err}`)
+      );
+  } else if (process.platform === 'darwin') {
+    addSymlink()
+      .then(() => logNotify(withNotification, 'Hyper CLI installed', `Symlink created at ${cliLinkPath}`))
+      .catch(err => {
+        // 'EINVAL' is returned by readlink,
+        // 'EEXIST' is returned by symlink
+        const error =
+          err.code === 'EEXIST' || err.code === 'EINVAL'
+            ? `File already exists: ${cliLinkPath}`
+            : `Symlink creation failed: ${err.code}`;
+
+        //eslint-disable-next-line no-console
+        console.error(err);
+        logNotify(withNotification, 'Hyper CLI installation failed', error);
+      });
+  } else {
+    withNotification &&
+      notify('Hyper CLI instalation', 'Command is added in PATH only at package installation. Please reinstall.');
```

Comment:
Should we open an issue about this?

Reply:
about what? Can you detail your thoughts?